src/helpers/assets.ts

```diff
@@ -34,11 +34,7 @@ const addEthPlaceholder = (
   const hasEth = !!ethereumUtils.getAccountAsset(ETH_ADDRESS);
 
   const { genericAssets } = store.getState().data;
-  if (
-    includePlaceholder &&
-    !hasEth &&
-    (assets.length > 0 || !emptyCollectibles)
-  ) {
+  if (includePlaceholder && !hasEth && assets.length > 0) {
```

Comment:
@estrattonbailey if this condition equals true, a placeholder row for ETH is added to the wallet's assets. currently, the placeholder ETH row is added if the user has no tokens but has NFTs. in this case, we want to replace this row with the ETH chart, so i'm removing the condition for `!emptyCollectibles` (aka user has no NFTs). now the placeholder row will only appear if the user has other non-ETH tokens.